tests/unit_tests/test_view_model.py

```diff
@@ -116,6 +118,5 @@ def test_older_done_items_includes_items_last_modified_late_yesterday():
     late_yesterday = now - timedelta(days=1, hours=hours_since_10_pm_yesterday)
     late_yesterday_item = done_item(1, late_yesterday)
     items = [late_yesterday_item]
-    view_model = ViewModel(items)
+    view_model = ViewModel(items, True)

```

Comment:
Lots of minor code changes - could it be worth the tradeoff to just set a default value on the class? Answer isn't necessarily yes, but worth considering!